backend/controllers/matchingController.js

```diff
@@ -25,16 +25,47 @@ router.get('/lists', auth, async (req, res) => {
 
         const role = response.data.role;
         const title = response.data.title;
+        const industry = response.data.industry;
+        const mentorship = response.data.mentorship;
 
-        const matchingRole = role === 'Mentee' ? 'Mentor' : role === 'Mentor' ? 'Mentee' : null;
+        const matchingRole = role.toLowerCase() === 'mentee' ? 'mentor' : role === 'mentor' ? 'mentee' : null;
```

Comment:
```
const role = response.data.role.toLowerCase();
const title = response.data.title.toLowerCase();
const industry = response.data.industry.toLowerCase();
const mentorship = response.data.mentorship.toLowerCase();

const matchingRole = role === 'mentee' ? 'mentor' : role === 'mentor' ? 'mentee' : null;
```

Reply:
This way you wouldnt have to change every time you re dealing with the string role

Reply:
That makes sense! Thanks.